scripts/diff.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+
+# 現在のブランチを取得
+current_branch=$(git branch --show-current)
+
+# Issue番号の特定（ブランチ名から抽出）
+issue=$(echo "$current_branch" | grep -oE 'GH-[0-9]+' | grep -oE '[0-9]+')
+
+# 基準ブランチ（デフォルトはmain）
+base_branch=${1:-"main"}
+
+# main ブランチを最新化
+if [ "$base_branch" = "main" ]; then
```

Comment:
![must-badge](https://img.shields.io/badge/review-must-red.svg)

ハードコーディングされているため、`base_branch`を参照するようにしてください。

Reply:
@r0227n 
以下の認識で問題ないでしょうか？

```suggestion
git fetch origin $base_branch:$base_branch 2>/dev/null || true
```

Reply:
@karas1910 
認識あいます！

Reply:
@r0227n 
[4c3da3d](https://github.com/yumemi-inc/flutter-mobile-project-template/pull/464/commits/4c3da3d8871f61182ff64aff78df3edf6cf9cef2) で修正しました！